app/models/review.rb

```diff
@@ -1,10 +1,10 @@
 class Review < ActiveRecord::Base
-  validates :rating, length: { in: 0..5 }, allow_blank: true
+  validates :rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }, allow_blank: true
   validates :rating, format: { with: /[+]?[0-5]{1}(\.\d)/ }, allow_blank: true
-  validates :safety_rating, length: { in: 0..5 }, numericality: { only_integer: true }, presence: true
-  validates :schools_rating, length: { in: 0..5 }, numericality: { only_integer: true }, presence: true
-  validates :food_entertainment, length: { in: 0..5 }, numericality: { only_integer: true }, presence: true
-  validates :public_transport, length: { in: 0..5 }, numericality: { only_integer: true }, presence: true
+  validates :safety_rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }, presence: true
+  validates :schools_rating, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }, presence: true
+  validates :food_entertainment, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }, presence: true
+  validates :public_transport, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }, presence: true
```

Comment:
I disagree with the hound, here - though you could try to break these up into more lines.